yearn/treasury/treasury.py

```diff
@@ -2,23 +2,26 @@
 import threading
 import time
 
-from brownie import Contract, chain, web3
+from brownie import ZERO_ADDRESS, Contract, chain, web3
 from brownie.network.event import EventLookupError
 from eth_abi import encode_single
 from eth_utils import encode_hex
 from joblib import Parallel, delayed
+from yearn.constants import (ERC20_TRANSFER_EVENT_HASH,
+                             ERC677_TRANSFER_EVENT_HASH, STRATEGIST_MULTISIG,
+                             TREASURY_WALLETS)
 from yearn.events import decode_logs
+from yearn.exceptions import PriceError
 from yearn.multicall2 import fetch_multicall
-from yearn.outputs import victoria
+from yearn.outputs.victoria import output_treasury
 from yearn.partners.partners import partners
 from yearn.partners.snapshot import WildcardWrapper, Wrapper
+from yearn.prices import compound
 from yearn.prices.constants import weth
-from yearn.prices.magic import get_price, logger as logger_price_magic
-from yearn.exceptions import PriceError
+from yearn.prices.magic import get_price
+from yearn.prices.magic import logger as logger_price_magic
 from yearn.utils import contract
 
-from yearn.constants import TREASURY_WALLETS, ERC20_TRANSFER_EVENT_HASH, ERC677_TRANSFER_EVENT_HASH
-
 logger = logging.getLogger(__name__)
 logger_price_magic.setLevel(logging.CRITICAL)
```

Comment:
i don't think it's appropriate to change log level of another module from outside a script or a debug session